content/en/docs/concepts/policy/pod-security-policy.md

```diff
@@ -609,11 +612,14 @@ are:
 - `unconfined` - Seccomp is not applied to the container processes (this is the
   default in Kubernetes), if no alternative is provided.
 - `runtime/default` - The default container runtime profile is used.
-- `docker/default` - The Docker default seccomp profile is used. Deprecated as of
-  Kubernetes 1.11. Use `runtime/default` instead.
+- `docker/default` - The Docker default seccomp profile is used. Deprecated as
+  of Kubernetes 1.11. Use `runtime/default` instead.
 - `localhost/<path>` - Specify a profile as a file on the node located at
   `<seccomp_root>/<path>`, where `<seccomp_root>` is defined via the
-  `--seccomp-profile-root` flag on the Kubelet.
+  `--seccomp-profile-root` flag on the Kubelet. (Note: this flag has been
+  deprecated in v1.19 and will be removed in v1.23. The seccomp root path will
+  then be derived from the kubelet root path, which is defined by `--root-dir`.
+  The current default value is `<root-dir>/seccomp`.)
```

Comment:
```suggestion
  `--seccomp-profile-root` flag on the kubelet.

   {{< note >}}
   Kubernetes v1.19 deprecated the `--seccomp-profile-root` flag, which will be removed in
   Kubernetes v1.23. The seccomp root path will   then be derived from the kubelet root path,
   which is defined by `--root-dir`. The current default value is `<root-dir>/seccomp`.
   {{< /note >}}
```
_or_, with [avoid statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future) in mind:
```suggestion
   the seccomp root path for that node. The default seccomp root path is a directory
   named `seccomp` directly beneath the kubelet's root directory (specified via the
   `--root-dir` command line flag to the kubelet).

   {{< note >}}
   The `--seccomp-profile-root` flag (deprecated since Kubernetes v1.19) allowed you to configure 
   the seccomp root path for a node. To use the defined behavior, leave that flag unset so that the
   kubelet uses its default.   
   {{< /note >}}
```

Reply:
@hasheddan , avoid statements/promises/predictions about the future.